tests/settings.py

```diff
@@ -21,6 +21,7 @@
     }
 }
 
+PASSWORD_HASHERS = ['django.contrib.auth.hashers.SHA1PasswordHasher']
```

Comment:
The passwords in the test fixtures are all old sha1 hashes which are no longer supported by default. This seemed like a better use of time than setting up a database and creating new users just to get new hashes.